<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInsertColumnDlg</class>
 <widget class="QDialog" name="CInsertColumnDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace Attribute Values</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="c_formula_desc">
         <property name="text">
          <string>Formula</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="c_formula"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Existing attributes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="c_column_names"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="c_selection_desc">
       <property name="text">
        <string>Apply formula to selected objects only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_use_column">
       <property name="text">
        <string>&lt;&lt; Use attribute</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_use_column</sender>
   <signal>clicked()</signal>
   <receiver>CInsertColumnDlg</receiver>
   <slot>OnUseAttribute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CInsertColumnDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CInsertColumnDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnUseAttribute()</slot>
  <slot>OnSelChangeColumnNames(QModelIndex)</slot>
  <slot>OnDblclkColumnNames(QModelIndex)</slot>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
